processing/src/main/java/io/druid/query/filter/NotDimFilter.java

```diff
@@ -69,6 +79,24 @@ public Filter toFilter()
   }
 
   @Override
+  public RangeSet<String> getDimensionRangeSet(String dimension)
+  {
+    if (field instanceof AndDimFilter) {
```

Comment:
Okay, that makes sense. Could you please include some comments for people like me that won't figure that out at first glance?